demisto_sdk/commands/validate/file_validator.py

```diff
@@ -812,30 +837,44 @@ def validate_all_files_schema(self):
             for dir_name in os.listdir(pack_path):
                 dir_path = os.path.join(pack_path, dir_name)
 
-                if dir_name not in CONTENT_ENTITIES_DIRS:
+                if dir_name not in CONTENT_ENTITIES_DIRS or \
+                        dir_name in [constants.REPORTS_DIR, constants.DASHBOARDS_DIR]:
```

Comment:
why?

Reply:
this is not a change - look at lines 829-830 on the previous file. it did exactly the same I just moved it to an easier location.

Reply:
@roysagi let's open an issue to fix this too please